web/src/assets/docs/DOCUMENTATION.md

```diff
@@ -143,25 +144,37 @@ or if you want to support us *louder*, you can use the large badge.
 [![nest badge](https://nest.land/badge-large.svg)](https://nest.land/package/your-package)
 ```
 
-# Updating your packages
-
-Updating your nest.land package versions is a breeze thanks to [@ebebbington](https://github.com/ebebbington)'s addition to the CLI.
+# Updating all of your dependencies
 
-Just type the command shown to automatically check for and update all nest.land package imports. Not only does this update nest.land packages, but it can also update [deno.land/std/](https://deno.land/std/) and [deno.land/x/](https://deno.land/x/) packages!
+Updating your dependencies versions is a breeze thanks to [our contributors](https://github.com/nestlandofficial/nest.land/graphs/contributors). Just type the command shown to automatically check for and update all module imports. Not only does this update nest.land modules, but it also can update [deno.land/std](https://deno.land/std/), [deno.land/x](https://deno.land/x/), [github](https://github.com/topics/deno) and [denopkg.com](https://denopkg.com/) modules!
```

Comment:
```suggestion
Updating your dependencies is a breeze thanks to [our contributors](https://github.com/nestlandofficial/nest.land/graphs/contributors). Just type the command shown to automatically check for and update all module imports. Not only does this update nest.land modules, but it can also update [deno.land/std](https://deno.land/std/), [deno.land/x](https://deno.land/x/), [github](https://github.com/topics/deno) and [denopkg.com](https://denopkg.com/) modules!
```